code/js/modules/BaseController.js

```diff
@@ -91,12 +114,20 @@
   };
 
   BaseController.prototype.playPause = function() {
-    if(this.selectors.play !== null && this.selectors.pause !== null) {
-      if(this.isPlaying()) {
+    if(this.play && this.pause) {
+
+      if(this.isPlaying())
+        this.pause();
+      else
+        this.play();
+
+    } else if(this.selectors.play !== null && this.selectors.pause !== null) {
+
+      if(this.isPlaying())
         this.click({action: "playPause", selectorButton: this.selectors.pause, selectorFrame: this.selectors.iframe});
-      } else {
+      else
```

Comment:
Mixed syntax. Every other conditional in this repo is
```javascript
if () { ... } else { ... }
```

Code quality requires consistency